picard/coverart/providers/caa.py

```diff
@@ -123,8 +72,97 @@ def caa_url_fallback_list(desired_size, thumbnails):
     return urls
 
 
+class arrow_button(QtWidgets.QPushButton):
```

Comment:
Class names should be camel-cased, so it should be `class ArrowButton`

Reply:
Fixed.